wemake_python_styleguide/visitors/ast/keywords.py

```diff
@@ -71,6 +80,11 @@ def _check_exception_type(self, node: ast.Raise) -> None:
                 BaseExceptionRaiseViolation(node, text=exception_name),
             )
 
+    def _check_bare_raise(self, node: ast.Raise) -> None:
+        parent = walk.get_closest_parent(node, ast.ExceptHandler)
+        if not parent:
```

Comment:
you also need to check  that `node.exc is None` to ensure that the given `raise` statement is bare